settings.gradle

```diff
@@ -89,5 +88,6 @@ include ':service-python'
 include ':service-python:assessclaimdc7101'
 include ':service-python:assessclaimdc7101v2'
 include ':service-python:assessclaimdc6602'
+include ':service-python:assessclaimdc6602v2'
```

Comment:
it seems that if the subproject is included here, it gets included in the build no matter if the prototype property is set

Reply:
I tried using this a as a conditional in settings.gradle but it did not work, `startParameter.projectProperties.hasProperty("enablePrototype")`, The only way I could figure out how to set a property in this file was through gradle.properties